code/controllers/subsystem/explosions.dm

```diff
@@ -459,16 +459,16 @@ This way we'll be able to draw the explosion's expansion path without having to
 		timer = TICK_USAGE_REAL
 		var/list/throw_turf = throwTurf
 		throwTurf = list()
-		for(var/t in throw_turf)
-			var/turf/affected_turf = t
+		for(var/turf/affected_turf AS in throw_turf)
 			if(QDELETED(affected_turf))
 				continue
-			for(var/am in affected_turf)
-				var/atom/movable/thing_to_throw = am
+			for(var/atom/movable/thing_to_throw AS in affected_turf)
```

Comment:
I realize it's the same as the old code functionally, but can turfs have non-AM contents that would justify worrying here?

Reply:
Can only be AM's